app/src/userpages/components/StreamPage/Edit/index.jsx

```diff
@@ -38,9 +39,17 @@ import styles from './edit.pcss'
 
 function StreamPageSidebar({ stream }) {
     const sidebar = useContext(SidebarContext)
+    const dispatch = useDispatch()
+
+    const streamId = stream && stream.id
+
     const onClose = useCallback(() => {
         sidebar.close()
-    }, [sidebar])
+
+        if (streamId) {
+            dispatch(resetResourcePermission('STREAM', streamId))
```

Comment:
Removing the permission after closing will trigger a reload on the list dropdown next time.